OxfordBodleian/Juel-Jensen/BDLaethe38.xml

```diff
@@ -136,51 +412,171 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <locus from="78vb" to="80rb"/>
+                                    <q xml:lang="gez">
+                                        <seg part="I">
+                                        <hi rend="rubric">ተአምረ፡ ዘገ<supplied reason="omitted">ብ</supplied>ረ፡</hi> እግዚእነ፡ ወአምላክነ፡ ኢየሱስ፡ ክርስቶ<supplied reason="omitted">ስ</supplied>
+                                        በኃይለ፡ ሞቱ፡ <sic>ማኀየዊ፡</sic> <add place="above">ይምሐሮ፡ ለ</add>ገብሩ፡ ገብረ፡ ማርያም፡ ወሀሎ፡ ፩ድውይ፡ እንዘ፡ ይግ<add place="above">ዕ</add>ር፡ 
+                                        ወየአውዩ፡ ፲<add place="above">፪</add>ዓመተ፡ ወዘረከቦ፡ በብዝኃ፡ ኃጢአቱ፡ ወተፈትነ፡ በእሳት፡ ከመ፡ ወርቅ፡ ወብሩር፡ መዓልተ፡ ወሌሊተ፡ </seg>
+                                    </q>
+                                    <note>
+                                        The note is hastily written. It narrates a miracle perfomed by Jesus Christ and mentions <persName ref="PRS12983GabraMa" role="patron"/>.
+                                        It might have been written in the same hand as that of <ref target="#a2"/>.
+                                    </note>
+                                </item>
+                                <item xml:id="a4">
+                                    <desc type="GuestText" xml:id="LIT5921SalamLaki"/>
+                                    <locus from="80va" to="80vb"/>
+                                    <q xml:lang="gez">
+                                        <seg part="I">
+                                            ሰላም፡ ለኪ፡ ደብረ፡ ሊባኖስ፡ ግነት፡ ፄና፡ መዓዛኪ፡ ዕፍረት፡ ዘአዕረፈ፡ ብኪ፡ ተክለ፡ <choice><sic>ሃማኖተ፡</sic><corr>ሃይማኖት፡</corr></choice>
+                                        </seg>
+                                    </q>
+                                   <note>Part of the text is hardly readable due to faded ink. Spaces for rubrication have been left unfilled.</note>
+                                </item>
+                                
+                                <item xml:id="e1">
+                                    <desc>A white sticker is glued to the inner side of the front board. The name of <persName ref="PRS5782JuelJen"/> is printed on it
+                                        (<foreign xml:lang="gez">ቤንት፡ ዩውል፡ የንሰን።</foreign>) and the signatures "MS. Aeth. e. 38" and "MS. Aethiop. J-J. 41"are written in
+                                        pencil. Below is glued an additional sticker, with the signature "Ms 41"printed on it.</desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <desc>Writings in pencil are drawn up on the recto side of the protective bifolium glued to the front cover.
+                                        <q xml:lang="en">81 folios. 10 full page paintings. 2 sketches. size 7" x 5"</q>.</desc>
+                                </item>
+                                <item xml:id="e3">
```

Comment:
here and in the following item, it would also be better to substitute note for desc